packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.10.2
+
+* Fixes typos in README and CHANGELOG.
```

Comment:
Nit: the typo is only in the README. There is no CHANGELOG fix in this package.

Reply:
Ok, I just ran `dart run script/tool/bin/flutter_plugin_tools.dart update-release-info --version=minimal --base-branch=upstream/main --changelog="Fixes typos in README and CHANGELOG."` I expected the change message to be put in both packages that were modified. I'm guessing the `update-release-info` tool is smarter and it ignores CHANGELOG changes?

Do you want me to:

a) Force an update to `google_maps_flutter_platform_interface`, or
b) Merely change "Fixes typos in README and CHANGELOG" above to "Fixes typo in README"?

Reply:
> I'm guessing the `update-release-info` tool is smarter and it ignores CHANGELOG changes?

Correct.

> b) Merely change "Fixes typos in README and CHANGELOG" above to "Fixes typo in README"?

This one. The trivial issue in the other CHANGELOG will be cleaned up on the next publish, and isn't worth releasing a new version for.

Reply:
Got it, and thanks for your patience with me. Change pushed now.